source/getting-started/robot-nervous-system/blackboard.md

```diff
@@ -91,6 +97,19 @@ let service = node.service_builder(&"global_config".try_into()?)
     .create()?;
 ```
 
+```{code-block} python
+battery_key = ctypes.c_uint64(0)
+us_sensor_key = ctypes.c_uint64(1)
+
+service = (
+    node.service_builder(iox2.ServiceName.new("My/Funk/ServiceName"))
```

Comment:
The service name should be "global_config". To be aligned with the documentation and the other code snippets.